go/vt/schemadiff/annotations.go

```diff
@@ -214,9 +214,16 @@ func unifiedAnnotated(from *TextualAnnotations, to *TextualAnnotations) *Textual
 // annotatedDiff returns the annotated representations of the from and to entities, and their unified representation.
 func annotatedDiff(diff EntityDiff, entityAnnotations *TextualAnnotations) (from *TextualAnnotations, to *TextualAnnotations, unified *TextualAnnotations) {
 	fromEntity, toEntity := diff.Entities()
+	// Handle the infamous golang interface is not-nil but underlying object is:
```

Comment:
TiL that an interface holding a nil value is not nil

Reply:
You're in for some fun times!